phpcs.xml

```diff
@@ -83,6 +83,7 @@
 
 		<exclude-pattern>templates/*.php</exclude-pattern>
 		<exclude-pattern>templates/**/*.php</exclude-pattern>
+		<exclude-pattern>uninstall.php</exclude-pattern>
```

Comment:
Why are we excluding this?

Reply:
I don't think we should exclude this in order to add a changelog to the fileheader. In this case we don't have a good precedent for how to record the changelog on the uninstall.php and I think we can just *not do it*